psydac/api/ast/fem.py

```diff
@@ -235,13 +236,18 @@ def __init__(self, expr, terminal_expr, spaces, tag=None):
         if isinstance(domain, Boundary):
             mask = Mask(domain.axis, domain.ext)
 
+        elif isinstance(domain, Interface):
+            mask = Mask(domain.axis, None)
+
         if isinstance(expr, LinearForm):
+
             is_linear           = True
             tests               = expr.test_functions
             fields              = expr.fields
             mapping             = spaces.symbolic_mapping
             is_rational_mapping = spaces.is_rational_mapping
             spaces              = spaces.symbolic_space
+            is_broken           = spaces.is_broken
```

Comment:
Does `is_broken` mean that the solution is not C0 across the interface between two patches?

Reply:
yes